src/index.html

```diff
@@ -26,9 +26,11 @@ <h1 class="page-heading">todos</h1>
         			</div>
                     <div class="clearer-container">
                         <button type="button" class="clear-completed-button">Clear completed</button>
-                    </div>        			
-        		</footer>
+                    </div>                           			
+        		</footer>                  
         	</div>
+            <div class="hidden-bar-like first"></div>
```

Comment:
When I said that this bottom sandwich is a shadow it meant that it is a shadow

.todo-bar-bottom:before {
	content: '';
	position: absolute;
	right: 0;
	bottom: 0;
	left: 0;
	height: 50px;
	overflow: hidden;
	box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2),
	            0 8px 0 -3px #f6f6f6,
	            0 9px 1px -3px rgba(0, 0, 0, 0.2),
	            0 16px 0 -6px #f6f6f6,
	            0 17px 2px -6px rgba(0, 0, 0, 0.2);
}